sample/src/main/java/com/github/theapache64/twyper/Util.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.theapache64.twyper
+
+import androidx.compose.ui.graphics.Color
+
+private val rgbRange = 0..255
+
+object Util {
+    fun randomColor(): Color {
```

Comment:
Looks unused. Do we need this method?

Reply:
I think you can remove this method, and then extract out the color generation logic from the `rememberRandomColor()`. Then you'll be able to use it from both composable and non-composable context.